application/src/main/java/com/github/pieter_groenendijk/repository/AccountRepository.java

```diff
@@ -0,0 +1,89 @@
+package com.github.pieter_groenendijk.repository;
+
+import java.util.Date;
+import java.util.UUID;
+import com.github.pieter_groenendijk.model.Account;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+import java.util.List;
+import com.github.pieter_groenendijk.hibernate.SessionFactoryFactory;
+
+public class AccountRepository implements IAccountRepository {
+
+    private SessionFactory sessionFactory;
+
+    public AccountRepository() {
+        sessionFactory = new SessionFactoryFactory().create();
+        System.out.println("Repo1");
+    }
+
+    public Account retrieveById(long id) {
+        Session session = sessionFactory.openSession();
+
+        Account account = null;
+        try {
+            session.beginTransaction();
```

Comment:
Lijkt mij dat er geen transaction nodig is voor een database read, of is dit een specifiek ORM ding?

Reply:
Nee, dat is niet nodig. Weggehaald.